lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -57,6 +57,7 @@ module Cardano.Wallet.Api.Server
     , getTransaction
     , listWallets
     , listStakeKeys
+    , listStakeKeys'
```

Comment:
I think some of the difficulties might be caused by how and where `listStakeKeys` and `listStakeKeys'` are split.
If it's possible to pass values, do that, rather than passing functions.